ui/src/app/components/copy-modal.tsx

```diff
@@ -21,54 +23,105 @@ const ResourceTypeHomeTabs = new Map()
   .set(ResourceType.DATASET, 'data');
 
 export interface Props {
+  cdrVersionListResponse: CdrVersionListResponse;
   fromWorkspaceNamespace: string;
-  fromWorkspaceName: string;
+  fromWorkspaceFCName: string;
   fromResourceName: string;
+  fromCdrVersionId: string;
   onClose: Function;
   onCopy: Function;
   resourceType: ResourceType;
   saveFunction: (CopyRequest) => Promise<FileDetail | ConceptSet>;
 }
 
 interface State {
-  writeableWorkspaces: Array<Workspace>;
+  workspaceOptions: Array<Object>;
   destination: Workspace;
   newName: string;
   requestState: RequestState;
-  errorMsg: string;
+  copyErrorMsg: string;
   loading: boolean;
 }
 
-const boldStyle = {
-  fontWeight: 600
-};
+const styles = reactStyles({
+  bold: {
+    fontWeight: 600
+  },
+});
 
-class CopyModal extends React.Component<Props, State> {
+class CopyModalComponent extends React.Component<Props, State> {
```

Comment:
renamed because this is no longer the primary exported artifact.  scroll down to the bottom of the file for the new `CopyModal`